sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/EncryptionKeyWrapMetadata.java

```diff
@@ -1,57 +1,53 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.cosmos.encryption;
+package com.azure.cosmos.models;
 
 import com.azure.cosmos.implementation.guava25.base.Preconditions;
+import com.azure.cosmos.util.Beta;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import java.util.Objects;
 
-import static com.azure.cosmos.implementation.encryption.ImplementationBridgeHelpers.EncryptionKeyWrapMetadataHelper;
-
-
 /**
  * Metadata that a key wrapping provider can use to wrap/unwrap data encryption keys.
- * {@link EncryptionKeyWrapProvider}
  */
-public class EncryptionKeyWrapMetadata {
+@Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+public final class EncryptionKeyWrapMetadata {
 
     /**
      * For JSON deserialize
      */
     EncryptionKeyWrapMetadata() {
     }
 
-    /**
-     * Creates a new instance of key wrap metadata.
-     *
-     * @param value Value of the metadata
-     */
-    public EncryptionKeyWrapMetadata(String value) {
-        this("custom", value);
-    }
-
     /**
      * Creates a new instance of key wrap metadata based on an existing instance.
      *
      * @param source Existing instance from which to initialize.
      */
+    @Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
     public EncryptionKeyWrapMetadata(EncryptionKeyWrapMetadata source) {
         this.type = source.type;
         this.algorithm = source.algorithm;
         this.value = source.value;
     }
 
-    EncryptionKeyWrapMetadata(String type, String value) {
-        this(type, value, null);
+    @Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public EncryptionKeyWrapMetadata(String name, String value) {
+        this("custom", name, value, null);
```

Comment:
>om", [](start = 18, length = 4)

We are making type mandatory in .net, can do the same here: [https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2283](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2283)

Reply:
Thanks , captured in miscellaneous encryption work item which will land after preview release. https://github.com/Azure/azure-sdk-for-java/issues/19990